drivers/pwm/pwm_mchp_xec_bbled.c

```diff
@@ -355,19 +425,24 @@ static int pwm_bbled_xec_init(const struct device *dev)
 		.girq_pos = (uint8_t)(DT_INST_PROP_BY_IDX(0, girqs, 1)),	\
 		.pcr_idx = (uint8_t)DT_INST_PROP_BY_IDX(inst, pcrs, 0),		\
 		.pcr_pos = (uint8_t)DT_INST_PROP_BY_IDX(inst, pcrs, 1),		\
-		.clk_sel = UTIL_CAT(XEC_PWM_BBLED_CLKSEL_, XEC_PWM_BBLED_CLKSEL(n)),	\
+		.clk_sel = UTIL_CAT(XEC_PWM_BBLED_CLKSEL_, XEC_PWM_BBLED_CLKSEL(inst)),	\
+		.enable_low_power_32K = (uint8_t)DT_INST_PROP_OR(inst, enable_low_power_32k, 0),\
```

Comment:
Boolean DTS properties are always defined, so using `_OR()` doesn't make sense here. Please just store the boolean in a `bool` variable without using `_OR()`.

Reply:
Updated with DT_INST_PROP macro